omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/ImportController.java

```diff
@@ -283,6 +283,11 @@ public String uploadPOST(UploadForm uploadForm, Errors errors, Model model) thro
 				importer.loadSerializedPackageStream(in);
 				in.close();
 			}
+			catch(Exception e){
+				log.error(e);
```

Comment:
thanks @Parth59  how do things look for the entire pr?

Reply:
I guess they should be good to go now !